### PR TITLE
Feature/ffms 583 manual management sale

### PR DESCRIPTION
**Pull request by LevioMike**

### CODE REVIEW

File: .DS_Store
Comment (thi-van-luong) [inline .DS_Store:1]: this file should not exist. some mac configuration created this file.

File: dal-identification.yml [hunk 1]
```diff
@@ -2955,23 +2955,23 @@ components:
         telephone:
           type: string
           maxLength: 10
-          pattern: '^\d{10}$'
+          #pattern: '^\d{10}$'
```
Comment (thi-van-luong) [inline dal-identification.yml:2958]: a phone number should be only numbers and no string. why did we deactivate this validation rule?

File: dal-identification.yml [hunk 2]
```diff
@@ -2955,23 +2955,23 @@ components:
...
         extension:
           type: string
           maxLength: 7
-          pattern: '^\d{0,7}$'
+          #pattern: '^\d{0,7}$'
```
Comment (thi-van-luong) [inline dal-identification.yml:2962]: an extension number should be only numbers and no string. why did we deactivate this validation rule?

File: dal-identification.yml [hunk 3]
```diff
@@ -2955,23 +2955,23 @@ components:
...
         cell:
           type: string
           maxLength: 10
-          pattern: '^\d{10}$'
+          #pattern: '^\d{10}$'
```
Comment (thi-van-luong) [inline dal-identification.yml:2966]: a phone number should be only numbers and no string. why did we deactivate this validation rule?

File: dal-identification.yml [hunk 4]
```diff
@@ -2955,23 +2955,23 @@ components:
...
         tollFree:
           type: string
           maxLength: 10
-          pattern: '^\d{10}$'
+          #pattern: '^\d{10}$'
```
Comment (thi-van-luong) [inline dal-identification.yml:2970]: a phone number should be only numbers and no string. why did we deactivate this validation rule?

File: dal-identification.yml [hunk 5]
```diff
@@ -2955,23 +2955,23 @@ components:
...
         fax:
           type: string
           maxLength: 10
-          pattern: '^\d{10}$'
+          #pattern: '^\d{10}$'
```
Comment (thi-van-luong) [inline dal-identification.yml:2974]: a fax number should be only numbers and no string. why did we deactivate this validation rule?

File: dal-identification.yml [hunk 6]
```diff
@@ -3957,4 +3960,53 @@ components:
           items:
             $ref: '#/components/schemas/AccessoryCharge'
         cubingRules:
-          $ref: '#/components/schemas/CubingRules'
\ No newline at end of file
+          $ref: '#/components/schemas/CubingRules'
+    Document:
```
Comment (thi-van-luong) [inline dal-identification.yml:3964]: this should not be here

File: dal-identification.yml [hunk 7]
```diff
@@ -3957,4 +3960,53 @@ components:
...
+          type: string
+          format: date
+        deleteTimestamp:
+          type: string
+          format: date
+        documentStatus:
+          $ref: '#/components/schemas/DocumentStatus'
+        documentType:
+          $ref: '#/components/schemas/DocumentType'
+    DocumentStatus:
```
Comment (thi-van-luong) [inline dal-identification.yml:4007]: this should not be here

File: dal-identification.yml [hunk 8]
```diff
@@ -3957,4 +3960,53 @@ components:
...
+      type: string
+      enum: [ INITIATE, SEND, ARCHIVED, ACCEPTED, DELETE ]
+    DocumentType:
```
Comment (thi-van-luong) [inline dal-identification.yml:4010]: this should not be here

**Review by thi-van-luong — CHANGES_REQUESTED** [review]

**Comment by thi-van-luong** [comment]
@LevioMike @franciskonchou view comments please.

File: dal-identification.yml (hunk 1 above)
Comment (LevioMike) [reply dal-identification.yml:2958]: @franciskonchou rappel-tu pourquoi tu as faite cette changement?

File: dal-identification.yml (hunk 2 above)
Comment (LevioMike) [reply dal-identification.yml:2962]: @franciskonchou rappel-tu pourquoi tu as faite cette changement?

File: dal-identification.yml (hunk 3 above)
Comment (LevioMike) [reply dal-identification.yml:2966]: idem

File: dal-identification.yml (hunk 4 above)
Comment (LevioMike) [reply dal-identification.yml:2970]: idem

File: dal-identification.yml (hunk 5 above)
Comment (LevioMike) [reply dal-identification.yml:2974]: idem @franciskonchou

File: dal-identification.yml (hunk 6 above)
Comment (LevioMike) [reply dal-identification.yml:3964]: @franciskonchou enleve et valide stp

File: dal-identification.yml (hunk 7 above)
Comment (LevioMike) [reply dal-identification.yml:4007]: idem

File: dal-identification.yml (hunk 8 above)
Comment (LevioMike) [reply dal-identification.yml:4010]: idem

**Pull request merged**